build/gulpfile.js

```diff
@@ -1 +1,63 @@
+var gulp = require("gulp");
+var eclint = require("eclint");
+var jsonlint = require("gulp-jsonlint");
+var zip = require("gulp-zip");
+var addsrc = require('gulp-add-src');
+var timestamp = require("./tasks/lib/timestamp");
+var transform = require('gulp-transform');
+var path = require("path");
+var _ = require('lodash');
+
 require('requiredir')('tasks', { recurse: true })
+
+_.mixin({
+
+  indexBy: function(list, func) {
+    for (var i = 0, l = list.length; i < l; i++) {
+      if (func(list[i])) return i;
+    }
+    return -1;
+  }
+
+});
```

Comment:
This doesn't look like it has the desired effect. In `patch-values.js`, the use of `indexBy` looks like it returns 0, [resulting in `dataSkuSettings` being set to `0`](https://github.com/elastic/azure-marketplace/pull/220/files#diff-f642ae29540d6d7b442c4eee187a2339R1032).

Is it needed only for patch-values.js?

Reply:
Updated to mixin `indexBy()` as alias for `_.keyBy()` per https://github.com/lodash/lodash/commit/b1d52ccd82caa374094b9c872883cdb622537a46

Reply:
👍 :100: